### PR TITLE
Add missing field check for scripted field in index pattern

### PR DESCRIPTION
**Pull request by sqren**
Closes elastic/kibana#26638

This PR fixes an issue that shows up in Discover in Kibana after importing the APM index pattern. 

screenshot 2018-12-04 at 13 44 59

The issue is caused by a change in ES 7.0 where missing fields will now throw:

> Starting in 7.0, doc['field'].value throws an exception if the field is missing in a document. To enable this behavior now, set a jvm.option -Des.scripting.exception_for_missing_value=true on a node. If you do not enable this behavior, a deprecation warning may be logged when painless processes a missing value.
https://www.elastic.co/guide/en/elasticsearch/painless/current/painless-examples.html#_missing_values

### CODE REVIEW

File: model/error/_meta/fields.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
     - name: error id icon
       type: keyword
-      script: "doc['error.grouping_key'].value"
+      script: "doc['error.grouping_key'].size() == 0 ? null : doc['error.grouping_key'].value"
```
Comment (sqren) [inline model/error/_meta/fields.yml]: Not sure if `null` is the best default value. LMK if you have better suggestions.

**Comment by sqren** [comment]
`include/fields.go: needs update`.

LMK what command needs running.

**Comment by graphaelli** [comment]
@sqren `make update` should regenerate that file

**Comment by sqren** [comment]
Thanks @graphaelli !

**Comment by sqren** [comment]
Jenkins retest please

**Comment by sqren** [comment]
Jenkins, retest this please

**Pull request merged**